exercises/largest-series-product/largest-series-product.example.ts

```diff
@@ -0,0 +1,52 @@
+export default class Series {
```

Comment:
A matter of opinion but I generally prefer writing 'pure' functions if I can. Input in and same output out every time. This all looks like it could be abstracted to take a more functional approach. Anyone have any thoughts here? I don't feel too strongly seeing as it is an example, although a functional example may be more understandable.

Reply:
I like functions too. I think it’s nice to put all related functions in a wrapper class even if this class doesn’t carry any state. It’s alittle cleaner this way.